### PR TITLE
Development

### PR DESCRIPTION
**Pull request by spotlighttriviaapp**

### CODE REVIEW

**Review by EvanCNavarro — APPROVED** [review]

**Review by EvanCNavarro — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ node.js: https://nodejs.org/en/download/
 firebase: npm install firebase
 
 # Workflow
-1. Create a branch off of development.
-2. Add your changes to your branch.
-3. Create a pull request into development.
+1. Checkout `development` branch.
+2. Add your changes.
+3. Push changes to github.
 4. When enough changes are accumulated in development that we want to go live, merge development branch into main.
```
Comment (EvanCNavarro) [inline readme.md:9]: I agree.

Comment (spotlighttriviaapp) [reply readme.md:9]: i like that you agree

**Review by EvanCNavarro — APPROVED** [review]

**Comment by patrick1bauer** [comment]
Looks good!

**Comment by EvanCNavarro** [comment]
PR approval test.

**Pull request merged**